lang/java/reef-io/src/test/java/org/apache/reef/io/data/loading/impl/EvaluatorRequestSerializerTest.java

```diff
@@ -24,6 +24,7 @@
 
 /**
  * Test serialization and deserialization of EvaluatorRequest objects.
+ * TODO[JIRA REEF-463]
```

Comment:
It meant "delete the test when deleting old serializer". But since we're deleting old serializer now, this is not needed